RNTester/README.md

```diff
@@ -8,21 +8,15 @@ Before running the app, make sure you ran:
 
     # in this folder
     npm install
-    # and in the toplevel folder
-    cd ..
-    npm install
 
 ### Running on iOS
 
 Mac OS and Xcode are required.
 
-- Start the packager with `npm start` **[important]**
 - Open a new tab in your terminal and start the bucklescript watcher with `npm run watch`
```

Comment:
This is the same for both platforms. I think it could be avoided with a post install script.